vendor/engines/secure_rooms/spec/features/scans_controller_spec.rb

```diff
@@ -0,0 +1,73 @@
+require "rails_helper"
+
+RSpec.describe "Scanning in and out", type: :controller do
```

Comment:
while it's not necessarily correct I usually expect anything under `spec/features` to use capybara type stuff, so I'd just keep it under `spec/controllers` for that. since it's in a `...Api` named module I'd probably look first in there for specs anyway

it's not too big a deal to me though